scripts/model_gen_ng/metaschema.yml

```diff
@@ -0,0 +1,168 @@
+"$schema": http://json-schema.org/draft-07/schema#
+"$ref": "#/definitions/Schema"
+definitions:
+  Schema:
+    additionalProperties: false
+    properties:
+      preamble:
+        type: string
+      types:
+        type: array
+        items:
+          "$ref": "#/definitions/Type"
+    required:
+      - types
+    type: object
+    title: Schema
+
+  Type:
+    oneOf:
+      - "$ref": "#/definitions/Enum"
+      - "$ref": "#/definitions/Struct"
+      - "$ref": "#/definitions/Custom"
+
+  # Enums
+  Enum:
+    properties:
+      name:
+        type: string
+      doc:
+        type: string
+      members:
+        type: array
+        items:
+          "$ref": "#/definitions/EnumMember"
+      type:
+        const: "enum"
+    additionalProperties: false
+    required:
+      - name
+      - type
+      - members
+    type: object
+
+  EnumMember:
+    properties:
+      name:
+        type: string
+      doc:
+        type: string
+    required:
+      - name
+    additionalItems: false
+
+  # Structs
+  Struct:
+    properties:
+      name:
+        type: string
+      doc:
+        type: string
+      fields:
+        type: array
+        items:
+          "$ref": "#/definitions/StructField"
+      type:
+        const: "struct"
+      key:
+        "$ref": "#/definitions/StructKey"
+      inherits:
+        type: string
+      tag:
+        type: string
+      default_comparable:
+        type: boolean
+        default: true
+      preamble:
+        type: string
+      virtual:
+        type: boolean
+    additionalProperties: false
+    required:
+      - name
+      - type
+      - fields
+    type: object
+
+  StructField:
+    properties:
+      name:
+        type: string
+      type:
+        type: string
+      default:
+        type: string
+      doc:
+        type: string
+    required:
+      - name
+    additionalItems: false
+
+  StructKey:
+    type: array
+    items:
+      type: string
+
+    additionalItems: false
+
+  BinaryArchive:
```

Comment:
Wat?

Reply:
Ops, I copypasted the jsonschema from orchestra to create this one and I accidentally left this snippet. Removed.